go.work

```diff
@@ -19,6 +19,6 @@ use (
 	./cmd/server
 	./common/archiver/gcloud
 
-	// DO NOT include, tools dependencies are intentionally separate.
-	// ./internal/tools
+// DO NOT include, tools dependencies are intentionally separate.
+// ./internal/tools
```

Comment:
`go work sync` on go 1.20 did this for some reason.

no idea why, but it wants it, and it seems harmless.